client/verta/tests/registry/conftest.py

```diff
@@ -17,4 +20,9 @@ def docker_image():
     )
 
 
-
+@pytest.fixture
+def make_model_schema_file(tmp_path, monkeypatch):
+    path = tmp_path / "model_schema.json"
+    schema = {"input": InputClass.schema()}
+    path.write_text(json.dumps(schema))
+    monkeypatch.setenv(_MODEL_SCHEMA_PATH_ENV_VAR, str(path))
```

Comment:
this is obviously not propagating but idk WHY (I've thrown a print in and verified it is in fact the default value of the env var):
```        except FileNotFoundError as e:
>           raise FileNotFoundError(
                "no schema found for model. Did you remember to `set_schema`?"
            ) from e
E           FileNotFoundError: no schema found for model. Did you remember to `set_schema`?
E           Falsifying example: test_validate_input_allow(
```